### DISCUSSION TITLE
A bug? Filter removes feeds that shouldn't be removed?

### DISCUSSION THREAD

**Discussion by turbodude** [martinrotter/rssguard#1424] [Q&A]
Hello, what am I missing? RSSGuard does receive news from from this feed with the filter enabled, but seemingly many items get filtered out that shouldn't be. Interestingly enough I noticed this by enabling notifications and seeing the deleted articles in the notification box.

The feed 
`https://www.real.discount/rss`

Feed item example
`https://www.real.discount/offer/essential-after-effects-from-beginner-to-motion-master-36861 Tue, 28 May 2024 13:01:01 GMT https://www.real.discount/offer/essential-after-effects-from-beginner-to-motion-master-36861 ]]> Published: 2024-02-23 12:16:07, Duration: 3.0 hours, Design Eye-Catching Motion Graphics with After Effects Even as a Beginner `

Page title
`[100% Off] Essential After Effects: From Beginner to Motion Master Free Course Coupon`

The filter:
```
var blacklist = [
'Python',
'Excel',
'Exams',
'Practice Tests',
'SQL',
'Cómo',
'Drupal',
'Introducción',
'Apprendre',
'Agile',
'HTML5',
'Experto',
'Angular',
'C++',
'Diploma',
'PHP',
'10Web',
'iOS',
'PowerPoint',
'Hindi',
'सी',
'खें'
];

function filterMessage() {
  if (blacklist.some(i => msg.title.indexOf(i) != -1)) {
    return MessageObject.Ignore;
  } else {
    return MessageObject.Accept;
  }
}
```

**Comment by turbodude** [comment]
Yeah something sketchy is going on here. It might not even be the filter, but RSSguard just skips some of the articles from that feed, although showing them in the notification balloon.

**Reply by martinrotter** [reply]
Provide full application log. On how to get it is described in documentation.
